src/Lecture1.hs

```diff
@@ -63,7 +68,10 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = if n < 0 
+    then (mod n (-10)) * (-1)
+    else mod n 10 
```

Comment:
🤯 I didn't know you can use `mod ... (-10)` to find the last digit of a negative number! That's neat 😎 

Your implementation is correct. You can write it shorter though by using the `abs` function that returns an absolute value of a number:

```haskell
λ: abs 7
7
λ: abs (-7)
7
```